recipes/lcaclassifier/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: lcaclassifier
+  version: 3.0.7
+
+source:
+  url: https://github.com/lanzen/CREST/releases/download/3.0.7/LCAClassifierV3.0.7.tar.gz
+  md5: ca072806b9251f280978c6a3ba520395
+  patches:
+    - lcaclassifier.patch
+
+build:
+  skip: True  # [py3k]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biom-format
```

Comment:
Only `python` and `setuptools` are required for the build; you can remove the other two.